cmd/gossamer/commands/account_test.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2023 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package commands
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// TestAccountGenerate test "gossamer account generate"
+func TestAccountGenerate(t *testing.T) {
+	testDir := t.TempDir()
+	directory := fmt.Sprintf("--keystore-path=%s", testDir)
+
+	AccountCmd.SetArgs([]string{"generate", directory})
```

Comment:
Should we copy the cmd by value, aren't we modifying the package variable for all other tests?

Can we add `t.Parallel()` to these tests?

Reply:
> Should we copy the cmd by value, aren't we modifying the package variable for all other tests?

I don't understand.

Reply:
Cannot add `t.Parallel()`. It messes up both cobra and viper for some reason